src/lambda/update-blog/index.py

```diff
@@ -0,0 +1,53 @@
+from boto3.s3.transfer import S3Transfer
+from datetime import datetime
+from util import *
+import boto3
+import logging
+import os
+
+s3 = boto3.client('s3')
+s3r = boto3.resource('s3')
+transfesr = S3Transfer(s3)
+
+dynamodb = boto3.resource('dynamodb')
+table    = dynamodb.Table(os.getenv('TABLE_NAME'))
+
+def insert_blog(file_path,key,bucket_name):
+    with open(file_path) as blog_content:
+        logging.info("Insert blog:" + key)
+        print(key)
+        try:
+            blog_content.readline()
```

Comment:
Hmm, Actually the frontmatter is usually written in yaml, this package might be helpful
[python-frontmatter](https://python-frontmatter.readthedocs.io/en/latest/)

Reply:
The python-frontmatter seems to work very well.